.travis.yml

```diff
@@ -1,13 +1,41 @@
 sudo: false
+
 language: python
+
 cache: pip 
+
 python:
   - 3.5
+
+env:
+  - PYDATA=true
+  - PYDATA=false CHECK_MIGRATION=true
+
+before_install:
+  - git config --global user.email "you@example.com"
+  - git config --global user.name "Your Name"
+  - if [[ $TRAVIS_PULL_REQUEST != false ]]; then
+      git pull -q --no-edit origin develop;
+    fi;
+  - if [[ $PYDATA == true ]]; then
+      git pull -q --no-edit origin deploy-pydata;
```

Comment:
> Adds pydata to INSTALLED_APPS and urlconfig. See the commit cf8603aa

Why not conditionally include it in the config? It could be controlled via envvar.

> Isn't pull an aggregate command that executes fetch and merge. Shouldn't be a problem.

Hmm well I can see some issues and questions here.

You're assuming that someone wants to merge into develop, which is not always true.

You're merging develop into PR branch (this may be the same as merging PR branch into develop - I'm not sure).

What happens when you get merge conflict. The test suite doesn't run?

And finally, according to the travis-ci [docs](https://docs.travis-ci.com/user/pull-requests):

> Rather than test the commits that have been pushed to the branch the pull request is from, we test the merge between the origin and the upstream branch. To only build on push events, you can disable Build on Pull Requests from your repository settings.

So… is this what @chrismedrela intended in #1066?